CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # **CHANGELOG**
 
+## **2.0.3**
+### ENHANCEMENTS
+- Support dedicated hosts for static worker nodes.
+- Support for Spectrum Scale storage nodes.
+
+### **BUG FIXES**
+- Fixed bug related to Http data source body depreciation.
```

Comment:
minor typo here... should be "deprecation"

Reply:
Fixed now